### PR TITLE
Staking

### PR DESCRIPTION
**Pull request by cd4761**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -420,14 +427,17 @@ def operator_create():
...
+        api_server = request.form['ApiServer']
 
         print("##################", pre_asset)
 
         if request.form["PreAsset"] == 'on':
             pre_asset = "true"
         else :
             pre_asset = "false"
-
+        print(rootchain_id)
```
Comment (ggs134) [inline app.py:440]: It should be cleaned up.

File: app.py [hunk 2]
```diff
@@ -397,6 +402,8 @@ def operator():
 @app.route("/operator/form")
 def operator_form():
     data = t_db.search(Query().Type == "rootchain")
+    data.append(config['NETWORKS']['MAINNET'])
+    print(data)
```
Comment (ggs134) [inline app.py:406]: It should be clean up.

File: app.py [hunk 3]
```diff
@@ -397,6 +402,8 @@ def operator():
 @app.route("/operator/form")
 def operator_form():
     data = t_db.search(Query().Type == "rootchain")
+    data.append(config['NETWORKS']['MAINNET'])
```
Comment (ggs134) [inline app.py:405]: It look like some `config.ini` field is changed. Where is README.md file which explains this field?

File: app.py (hunk 1 above)
Comment (cd4761) [reply app.py:440]: i'll do it

File: app.py (hunk 3 above)
Comment (cd4761) [reply app.py]: It was my mistake. I'm considering how to insert Mainnet and Rinkeby endpoint data to database.

**Pull request merged**